### DISCUSSION TITLE
Is there any command to print only installed AUR packages?

### DISCUSSION THREAD

**Discussion by julkefol** [Morganamilo/paru#651] [General]
As the title says.

**Comment by balroggg** [comment]
Maybe `paru -Qqem` what you want.

**Reply by julkefol** [reply]
What do the '-e' and '-m' flags do?

**Reply by balroggg** [reply]
It's `pacman` flags, your can check `man pacman`.
```
-e, --explicit
           Restrict or filter output to explicitly installed packages.
-m, --foreign
           Restrict or filter output to packages that were not found in the sync database(s).
```
`-em` filter explicitly installed packages not found in the sync database(s).

**Reply by balroggg** [reply]
Be aware `-e` filter aur dependancy so `paru -Qqem` != `pacman -Qq | paru -Ta -`

**Comment by Morganamilo** [comment]
There's also `pacman -Qq | paru -Ta -` depending on what you actually need.

**Reply by julkefol** [reply]
Thank you. But I'm preferring `paru -Qem` because it's faster imo.
